src/Sdk/WebApi/WebApi/ResultsHttpClient.cs

```diff
@@ -422,6 +514,9 @@ public static class Constants
         public static readonly string ResultsProtoApiV1Endpoint = "twirp/github.actions.results.api.v1.WorkflowStepUpdateService/";
         public static readonly string WorkflowStepsUpdate = ResultsProtoApiV1Endpoint + "WorkflowStepsUpdate";
 
+        public static readonly int DefaultNetworkTimeoutInSeconds = 30;
+        public static readonly int DefaultBlobUploadRetries = 3;
```

Comment:
do you want to hard code these in the runner or pass it down from the service as variable?

Reply:
it depends on whether you plan to tweak these in the future.

Reply:
Hard code those seems okay.  We have 100s timeout hardcoded before this as well.  

Maybe we could override those with env variables, passing down from server maybe a bit overkill?

Reply:
i am fine with either way, it's up to you. 😄

Reply:
I am going to leave this at 30s for now to collect more data.  We can always revert back to the current behavior with the feature flag.  Since this is only used to upload logs and summaries, I think 30s should be plenty time.